File: Arcade/Intro/27_variableName_v1.js
function variableName(name) {
    // Check if first character is a digit
    if (name.charAt(0).match(/[0-9]/)) {
        return false;
    }
    // Check if there is a character that isn't a letter, digit, or _
    if (name.match(/[^a-zA-Z0-9_]/)) {
        return false;
    }
    
    return true;
}

File: Arcade/Intro/C++/27_variableName_v1.cpp
bool variableName(std::string name) {
    if(name[0]>='0' && name[0]<='9') return false;
    for(char c:name){
        if(!(((c>='a'&&c<='z')||(c>='A'&&c<='Z'))||(c>='0'&&c<='9')||(c=='_'))) return false;
    }
           return true;
}
